design/noobaa/upgrades.md

```diff
@@ -0,0 +1,4 @@
+♜ [Rook NooBaa Design](README.md) /
+# Upgrades
+
+TODO
```

Comment:
Versioning question... Will the version of Noobaa will be included in the rook-noobaa operator image, or will a separate image will be deployed for the noobaa daemons? For example, originally rook included ceph in its own image and the versions were tied together, but in the 0.9 release we changed it to decouple the rook version from the ceph version. I'm assuming you don't need that flexibility for now, but wanted to ask the question.

Reply:
I saw that cockroach and minio and edgefs operators used that model where the operator image is based on an image of the target pod, and that also serves as the version selection when the operator starts and reads it's own image name. However, the Cassandra operator just uses Alpine base and chooses the deployment image differently.

I don't think this makes that much sense and I plan to just use the same model you suggested, like the Cassandra operator.

Let me know if there's an additional action item on that.
Resolving.